### DISCUSSION TITLE
SemVer implementation that's compatible

### DISCUSSION THREAD

**Discussion by 0b1kn00b** [babashka/babashka#1739] [General]
This is the first one I found with a  `deps.edn` available
[here](https://github.com/dotinspace/semver)

**Comment by borkdude** [comment]
Is this a question or are you sharing some information? I don't see a question mark anywhere

**Reply by 0b1kn00b** [reply]
I don't know if there's a postbox for stuff to add to the projects section so I came here.

**Reply by borkdude** [reply]
Ah great, thank you.

We can include it here: 

https://babashka.org/toolbox/ 
https://github.com/babashka/toolbox

**Reply by borkdude** [reply]
I added the library there.
